src/remote/remote.cpp

```diff
@@ -1425,7 +1425,13 @@ bool REMOTE_inflate(rem_port* port, PacketReceive* packet_receive, UCHAR* buffer
 {
 #ifdef WIRE_COMPRESS_SUPPORT
 	if (!port->port_compressed)
+	{
 		return packet_receive(port, buffer, buffer_length, length);
+	}//if
+
+	fb_assert(port->port_compressed);
+
+	// Attention: PORT_compression flag may still not setup!
```

Comment:
All this block does not look good.
1. Adding { } for single line block does not follow FB code standard (both styles possible but no cleanup from one to other).
2. "assert(expr)" right after "if (!expr) return" look wierd
3. Syntax error in comment, not PORT_compressION but PORT_compressED